### PR TITLE
Bump crates versions

### PR DESCRIPTION
**Pull request by nieznanysprawiciel**

### CODE REVIEW

**Review by mfranciszkiewicz — APPROVED** [review]

**Review by tworec — APPROVED** [review]

File: crates/util/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ya-sb-util"
-version = "0.1.0"
+version = "0.2.0"
```
Comment (nieznanysprawiciel) [inline crates/util/Cargo.toml]:
```suggestion
version = "0.1.0"
```

File: crates/router/Cargo.toml [hunk 2]
```diff
@@ -22,8 +22,8 @@ path = "src/bin.rs"
 required-features=['bin']
 
 [dependencies]
-ya-sb-proto = { version = "0.1", path = "../proto" }
-ya-sb-util = { version = "0.1", path = "../util" }
+ya-sb-proto = { version = "0.2", path = "../proto" }
+ya-sb-util = { version = "0.2", path = "../util" }
```
Comment (nieznanysprawiciel) [inline crates/router/Cargo.toml]:
```suggestion
ya-sb-util = { version = "0.1", path = "../util" }
```

File: Cargo.toml [hunk 3]
```diff
@@ -15,8 +15,8 @@ flex = ["flexbuffers"]
 json = ["serde_json"]
 
 [dependencies]
-ya-sb-proto = { version = "0.1", path = "crates/proto" }
-ya-sb-util = { version = "0.1", features=["with-futures"], path = "crates/util" }
+ya-sb-proto = { version = "0.2", path = "crates/proto" }
+ya-sb-util = { version = "0.2", features=["with-futures"], path = "crates/util" }
```
Comment (nieznanysprawiciel) [inline Cargo.toml]:
```suggestion
ya-sb-util = { version = "0.1", features=["with-futures"], path = "crates/util" }
```

**Comment by mfranciszkiewicz** [comment]
no changes in `ya-sb-derive`

**Pull request closed without merging**